test/functional-api/integration/callouts/callouts.request.params.ts

```diff
@@ -0,0 +1,53 @@
+import { TestUser } from '@test/utils';
+import { calloutData } from '@test/utils/common-params';
+import { graphqlRequestAuth } from '@test/utils/graphql.request';
+
+export enum CalloutState {
```

Comment:
Put the enums in a separate file, assuming you'll need them in multiple places. Doesn't make sense to search them in request params file from external file.